website/photos/api/v2/views.py

```diff
@@ -51,6 +55,36 @@ def get_queryset(self):
         )
 
 
+class LikedPhotosListView(ListAPIView):
+    """Returns the details the liked album."""
+
+    serializer_class = PhotoListSerializer
+    permission_classes = [
+        IsAuthenticatedOrTokenHasScope,
+    ]
+    required_scopes = ["photos:read"]
+
+    def get_queryset(self):
+        if self.request.member:
```

Comment:
I would not put this check here, but in a different method, such as `dispatch`, `get` or maybe use a permission class for this. Also, non-members should not get a 400, because it's not a bad request in itself (it's not malformed or something like that). I think a 403 would be better